RetroBar/Languages/Deutsch.xaml

```diff
@@ -40,25 +66,40 @@
 
     <s:String x:Key="start_text">Start</s:String>
     <s:String x:Key="start_text_xp">Start</s:String>
-    <s:String x:Key="start_button_tip_98">Klicken Sie zum Starten auf diese Schaltfläche.</s:String>
-    <s:String x:Key="start_button_tip">Klicken Sie zum Starten auf diese Schaltfläche.</s:String>
+    <s:String x:Key="start_button_tip_98">Hier klicken, um zu beginnen.</s:String>
+    <s:String x:Key="start_button_tip">Hier klicken, um zu beginnen</s:String>
```

Comment:
Were these strings taken from the source? This is the tooltip for the Start button.
Always try to use the original strings for a more faithful experience.

Reply:
The original (english source) contains on that two strings:
"Click here to begin."
"Click here to begin"

So the current translation is incorrect:
"Klicken Sie zum Starten auf diese Schaltfläche."
"Klicken Sie zum Starten auf diese Schaltfläche."

In English = "Click this button to start."

Reply:
Sure, but that's how it was originally translated for Windows, it doesn't have to be a literal translation.

Reply:
In cases where there is a difference between the translation in Windows versus the literal translation from English, we should use the same translation as Windows.

Reply:
It's what I went with as well, trying to come up with your own solutions may not always be 100% (even if the original translation also wasn't 100%)

Reply:
I've looked into it in Windows XP and changed the translation back, because its the original text from it.
Since Windows 7 the tooltip for the start button contains only "Start".